tests/python/gaia-ui-tests/requirements.txt

```diff
@@ -1,5 +1,5 @@
 boto>=2.32.1
-marionette_driver>=0.9
-marionette_client>=0.17
+marionette_driver==0.13
+marionette_client==0.17
 requests
 treeherder-client==1.5
```

Comment:
ugh, it was accidentally included, it won't run without this at the time, i'll clean it up.